packages/ckeditor5-package-generator/lib/templates/ts/package.json

```
@@ -80,14 +81,16 @@
     "typescript": "^4.7.4"
   },
   "scripts": {
+    "build": "tsc -p ./tsconfig.release.json",
```

```suggestion
    "ts:build": "tsc -p ./tsconfig.release.json",
```

We have two building scripts (DLL and TS), so let's add a prefix.